usr/libexec/greenboot/greenboot-service-monitor

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+GREENBOOT_CONFIGURATION_FILE=/etc/greenboot/greenboot.conf
+service_started=true
+if test -f "$GREENBOOT_CONFIGURATION_FILE"; then
+    source $GREENBOOT_CONFIGURATION_FILE
+fi
+
+if [ -n "$GREENBOOT_MONITOR_SERVICES" ]; then
+    services=($GREENBOOT_MONITOR_SERVICES)
+else
+    echo "GREENBOOT_MONITOR_SERVICE not found, using default value"
+    services=( sshd NetworkManager )
```

Comment:
I still think we have to drop this, and then any distribution can just set a default in the /etc config they ship ;)

Reply:
+1 to just setting the default services in the config that's shipped instead of hardcoding it here